### PR TITLE
Back on the train

### PR DESCRIPTION
**Pull request by misterybodon**
There are a few small changes, I've tested, no breakings. Do you prefer pull requests to a particular branch? If the github page is set from the master, it'd be better to pull to a different one. But you probably know this already...

Anyways, there we go.

ps: please anyone could add stephen hawking speeding across :-D

### CODE REVIEW

File: assets/css/challenge1.css [hunk 1]
```diff
@@ -111,9 +111,23 @@ div#theTradeCoder{
...
+    border: 2px #b35858 solid;
+    box-shadow: 1px 1px 1px 1px rgba(0,0,0,0.7), 2px 2px 2px 2px rgba(0,0,0,0.3);
+}
+#misterybodon h1{color:rgba(150,150,150);
+font-family: "Comic Sans MS", cursive, sans-serif;
+font-size:1.8rem;
+padding:0.5rem;
+}
+#misterybodon img{
+clip-path: polygon(50% 0%, 80% 10%, 100% 35%, 100% 56%, 66% 60%, 48% 100%, 33% 61%, 0 58%, 0% 35%, 20% 10%);
```
Comment (DavidMatthewFraser) [inline assets/css/challenge1.css:127]: I love the theme 👍
The umbrella is dope!

File: challenges/challenge2.html [hunk 2]
```diff
@@ -43,6 +43,7 @@ <h1 id='title' class='text-center p-4'>Open-Source Solar System</h1>
       <img id='davids_spaceship' src='../assets/img/spaceship.gif' class=''>
       <img id='biscuit_ufo' src='../assets/img/ufo.gif'>
       <img id="terris_comet" src="../assets/img/comet.png">
+      <img id='nobody_se' src='../assets/img/se.png'/>
```
Comment (DavidMatthewFraser) [inline challenges/challenge2.html:46]: Awesome! Love the animation!

File: challenges/challenge1.html [hunk 3]
```diff
@@ -111,6 +111,13 @@ <h1>Terri Fricker</h1>
             <p><q>Stop trying to make a point and focus on making a difference.</q> - Scott Martin</p>
             </div>
 
+            <!-- avatar for Mister Nobody -->
+            <div class="camperCell col-md-5 col-lg-3" id="misterybodon">
+            <h1>Mister Nobody</h1>
+            <img src="https://i.imgur.com/ws5rzqB.jpg"
+               alt="Avatar of Nobody">
+            <p><q cite="https://www.gutenberg.org/files/45315/45315-h/45315-h.htm">The nakedness of woman is the work of God.</q><br>&dagger; William Blake &dagger;</p>
```
Comment (DavidMatthewFraser) [inline challenges/challenge1.html:119]: William Blake is a cool dude!
I just think that content on our page should be more PG. What do you think?

**Comment by misterybodon** [comment]
Hey Sr!
Aha, *maybe *(though, honestly, it's nice to be a bit out of the rule), and
he says it in an almost religious way.

But I could find another one, don't take it bad :-)

**Comment by misterybodon** [comment]
Also, don't feel like you have to say nice things every time, just say hey
ok/not ok, and it's fine for me.

**Comment by DavidMatthewFraser** [comment]
Yea thanks, I was just worried someone wouldn't the context of the quote

**Comment by DavidMatthewFraser** [comment]
ok/not ok it is

**Comment by SyntappZ** [comment]
@misterybodon dont be putting your religion ads on the site man, this is not for that, this is for people who want to learn to code, not to read about heaven and hell. people have different views

**Comment by santimirandarp** [comment]
> @misterybodon dont be putting your religion ads on the site man, this is not for that, this is for people who want to learn to code, not to read about heaven and hell. people have different views

what's the part you don't like?

**Comment by misterybodon** [comment]
@SyntappZ that's a funny comment. It will take you a couple of years to change that comment, but hopefully you will.

**Comment by misterybodon** [comment]
> Could you change the quote? I was just worried someone wouldn't understand the context and get offended. If you don't think this is fair we can further discuss it or get a second opinion.

Yes general, I'm on the way.

**Comment by SyntappZ** [comment]
@misterybodon  the fuck ya talking about man

**Comment by SyntappZ** [comment]
@misterybodon funny man, i like ya attitude, i get your change now on stephen hawking, but im from a coucil estate ya little clown so that remark effected me 0, dont judge people by there covers ygm,

**Comment by DavidMatthewFraser** [comment]
@misterybodon PRs to master are just fine :)

**Comment by SyntappZ** [comment]
you forgot to remove his ad

**Pull request merged**